pkgs/applications/audio/wolf-spectrum/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitHub
+, pkg-config
+, libjack2
+, lv2
+, xorg
+, liblo
+, libGL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wolf-spectrum";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "pdesaulniers";
+    repo = "wolf-spectrum";
+    rev = "v${version}";
+    sha256 = "17db1jlj7vb1xyvkdhhrsvdbwb7jqw6i4168cdvlj3yvn2ra8gpm";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libjack2 lv2 xorg.libX11 liblo libGL xorg.libXcursor ];
+
+  makeFlags =
+    [ "BUILD_LV2=true" "BUILD_DSSI=false" "BUILD_VST2=true" "BUILD_JACK=true" ];
+
+  patchPhase = ''
+    patchShebangs ./dpf/utils/generate-ttl.sh
+  '';
+
+  installPhase = ''
+    mkdir -p $out/lib/lv2
+    #mkdir -p $out/lib/dssi
```

Comment:
Randomly commented-out stuff, and this `installPhase` in general: Please add some comments that describe what's (not) copied and why.

I suspect from `"BUILD_DSSI=false"` above that it doesn't build and you might want it to build in the future -- a comment on `"BUILD_DSSI=false"` might be excellent for that (you may also want to make the `makeFlags` above multi-line to best be able to place your comment).